config/memory_statistics.py

```diff
@@ -0,0 +1,114 @@
+import click
+from swsscommon.swsscommon import ConfigDBConnector
+# from utilities_common.cli import AbbreviationGroup
+
+
+class AbbreviationGroup(click.Group):
+    def get_command(self, ctx, cmd_name):
+        return super().get_command(ctx, cmd_name)
+
+
+@click.group(cls=AbbreviationGroup, name="memory-statistics")
+def memory_statistics():
+    """Configure the Memory Statistics feature"""
+    pass
+
+
+def check_memory_statistics_table_existence(memory_statistics_table):
+    """Checks whether the 'MEMORY_STATISTICS' table is configured in Config DB."""
+    if not memory_statistics_table:
+        click.echo("Unable to retrieve 'MEMORY_STATISTICS' table from Config DB.", err=True)
+        return False
+
+    if "memory_statistics" not in memory_statistics_table:
```

Comment:
>memory_statistics [](http://example.com/codeflow?start=8&length=17)

Define a constant?

Reply:
The memory_statistics key is now defined as a constant, improving code readability and maintainability.